lib/util.js

```diff
@@ -159,9 +159,9 @@ function stylizeNoColor(str, styleType) {
 
 
 function arrayToHash(array) {
-  var hash = {};
+  var hash = Object.create(null);;
 
-  array.forEach(function(val, idx) {
+  array.forEach(function(val) {
```

Comment:
Would we want to continue to use arrow functions for inline anons?

```js
array.forEach(val => hash[val] = true);
```

and/or convert this into a single reduce expression?

```js
function arrayToHash(array) {
  return array.reduce((acc, val) => {
    acc[val] = true;
    return acc;
  }, Object.create(null));
}
```